src/components/AbsenceList/index.tsx

```diff
@@ -10,13 +10,22 @@ const AbsenceList = () => {
 
   return (
     <div>
-      <ul className="list-none">
-        {data?.map((absence) => (
-          <li className="py-3" key={absence.id}>
+      <table className="w-full">
+        <thead>
+          <tr className="flex justify-start gap-10 text-left">
+            <th className="w-1/5">Name</th>
+            <th className="w-1/5">Absence Type</th>
+            <th className="w-1/5">Start Date</th>
+            <th className="w-1/5">End Date</th>
+            <th className="w-1/5">Approval Status</th>
+          </tr>
+        </thead>
+        <tbody>
+          {data?.map((absence) => (
             <AbsenceItem absence={absence} />
```

Comment:
should stick a key on the parent element when mapping out things in react. Should be able to work if you just stick it as a prop here, and you shouldn't need to handle that